canonical-web-and-design/maas-ui
# refactor(ui): move machine storage table components to node folder

## Done

- Moved machine storage table components to base node components folder
- Created wrapper `StorageTables` component which will eventually work for both machines and controllers

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Check that you can still edit storage for Ready or Allocated machines
- Check that other machines render storage details and are not editable

Fixes canonical-web-and-design/app-tribe#994
